internal/pkg/skuba/addons/addons.go

```diff
@@ -66,6 +67,8 @@ type AddonConfiguration struct {
 
 type addonTemplater func(AddonConfiguration) string
 
+type getImageCb func(imageTag string) string
```

Comment:
Nitpick: I would rename it to `getImageCallback`, since we are using the full `callback` word instead of `cb` in other places.

Reply:
+1

Reply:
Renamed :)